packages/@okta/i18n/src/properties/login.properties

```
@@ -1193,6 +1193,35 @@ oie.user.profile.lastname= Last name
 oie.user.profile.firstname= First name
 oie.user.profile.primary.email= Email
 oie.user.profile.secondary.email= Secondary email
+oie.user.profile.middleName = Middle name
+oie.user.profile.honorificPrefix = Honorific prefix
+oie.user.profile.honorificSuffix = Honorific suffix
+oie.user.profile.primaryEmail = Primary email
+oie.user.profile.title = Title
+oie.user.profile.displayName = Display name
+oie.user.profile.nickName = Nickname
+oie.user.profile.profileUrl = Profile URL
+oie.user.profile.secondEmail = Secondary email
+oie.user.profile.mobilePhone = Mobile phone
+oie.user.profile.primaryPhone = Primary phone
+oie.user.profile.streetAddress = Street address
+oie.user.profile.city = City
+oie.user.profile.state = State
+oie.user.profile.zipCode = ZIP Code
```

Can we confirm that we're using the traditional trademark styling by capitalizing `Code` here as well?

IMO we should be consistent and keep this lowercase:

```suggestion
oie.user.profile.zipCode = ZIP code
```

---

For this term, we inherit from the Microsoft Manual of Style: https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/z/zip-code
It should be "ZIP Code"

---

Thanks @IanHakes-okta! Is this the style we generally try to follow throughout all Okta products?

---

Yes - at least it should be! :-) 
(InfoDev is currently looking to hire a UI copywriter who will be responsible for making sure the strings get  standardized across the UI, but in the absence of that person, we're trying to make sure we use the style guidance we already have for anything new.)